<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusToolWid</class>
 <widget class="QWidget" name="StatusToolWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="loopThresholdBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>326</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>回路阈值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputThresholdBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>383</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>单元阈值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="thresholdBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>相位阈值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loopBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>回路信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>213</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>传输统计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>157</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>单元信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="devBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>43</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>相位信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="envBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>环境信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
